.gitignore

```diff
@@ -0,0 +1,3 @@
+build
+install
+log
```

Comment:
this is concerning

Reply:
this just allows for building the underlay without git commiting the colcon files. This only applies to the WLD dir, so I don't see why this would be bad.